### PR TITLE
[DEV-2570] dfx services login integration

### PR DESCRIPTION
**Pull request by lapatric**
- [ ] **Update Github secrets**

### CODE REVIEW

File: src/hooks/metamask.hook.ts
Comment (davidleomay) [inline src/hooks/metamask.hook.ts:1]: Tests seem to fail

File: src/index.tsx [hunk 1]
```diff
@@ -2,6 +2,9 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import * as buffer from 'buffer';
+
+window.Buffer = buffer.Buffer;
```
[the remark this answers is not in the document]
Comment (lapatric) [reply src/index.tsx:7]: Got a `ReferenceError: Buffer is not defined` when testing the hardware wallet connect in the app. The error was pointing to some line in the @ledgerhq package. The solution is taken from [here](https://solana.stackexchange.com/questions/212/uncaught-referenceerror-buffer-is-not-defined-using-phantom-wallet-solana-and).

[the remark this answers is not in the document]
Comment (davidleomay) [reply src/index.tsx:7]: Interesting 🤔

[the remark this answers is not in the document]
Comment (davidleomay) [reply src/index.tsx:7]: Ah, looks like we solved it with `config-overrides.js` in the Services: https://github.com/DFXswiss/services/commit/179dca22c0c12b3bc4f6713764cd85f1cd71db30

[the remark this answers is not in the document]
Comment (lapatric) [reply src/index.tsx:7]: Oh I see, will use the same approach here in that case 👍

**Pull request closed without merging**